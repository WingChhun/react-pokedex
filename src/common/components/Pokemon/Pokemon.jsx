import React, { memo } from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';



function Pokemon(props) {
    return (
        <div>
            POKEMON
        </div>
    );
}

Pokemon.propTypes = {};
Pokemon.defaultProps = {};

export default memo(Pokemon);