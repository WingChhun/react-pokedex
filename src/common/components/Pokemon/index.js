export { default as PokemonList } from './PokemonList';
export { default as PokemonCard } from './PokemonCard';
