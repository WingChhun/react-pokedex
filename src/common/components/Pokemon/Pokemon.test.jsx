import React from 'react';
import Pokemon from "./Pokemon";
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() })


//todo: test hooks
describe(`<Pokemon/>`, () => {

    it(`renders properly`, () => {
        const wrapper = shallow(<Pokemon />);
        expect(wrapper).toMatchSnapshot();
    })
});