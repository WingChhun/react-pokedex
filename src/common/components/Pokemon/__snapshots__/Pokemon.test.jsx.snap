// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pokemon/> renders properly 1`] = `ShallowWrapper {}`;
